frontend/public/components/monitoring/alerting.tsx

```diff
@@ -282,17 +282,28 @@ const PopoverField: React.FC<{ body: React.ReactNode; label: string }> = ({ body
 const alertStateHelp = (
   <dl className="co-inline">
     <dt>
-      <AlertStateIcon state={AlertStates.Firing} /> <strong>Firing: </strong>
+      <AlertStateIcon state={AlertStates.Pending} /> <strong>Pending: </strong>
     </dt>
-    <dd>The alert condition is true for the duration of the timeout.</dd>
+    <dd>
+      The alert is active but is waiting for the duration that is specified in the alerting rule
+      before it fires.
+    </dd>
     <dt>
-      <AlertStateIcon state={AlertStates.Pending} /> <strong>Pending: </strong>
+      <AlertStateIcon state={AlertStates.Firing} /> <strong>Firing: </strong>
     </dt>
-    <dd>The alert condition is true, but the timeout has not been reached.</dd>
+    <dd>
+      The alert is firing because the alert condition is true and the optional `for` duration has
+      passed. The alert will continue to fire as long as the condition remains true.
+    </dd>
     <dt>
       <AlertStateIcon state={AlertStates.Silenced} /> <strong>Silenced: </strong>
     </dt>
-    <dd>The alert is now silenced.</dd>
+    <dt></dt>
```

Comment:
Did you mean to add an empty `dt`?

Reply:
Yeah, I checked and technically, HTML5 doesn't allow `dd` without a `dt`.

https://html.spec.whatwg.org/#the-dd-element says `dd` can be used:
> After `dt` or `dd` elements inside `dl` elements.

Also, https://html.spec.whatwg.org/#the-dl-element says that `dl` contains:
> Zero or more groups each consisting of one or more `dt` elements followed by one or more `dd` elements...